lib/travis/build/git.rb

```diff
@@ -7,7 +7,7 @@ module Travis
   module Build
     class Git
       DEFAULTS = {
-        git: { depth: 50, submodules: true, strategy: 'clone', quiet: false }
+        git: { depth: 50, submodules: true, strategy: 'clone', quiet: false, sparseCheckout: false }
```

Comment:
Casing should be `sparse_checkout`. I understand the `git` option is camel case, but here, it's Ruby.